src/journal/journalctl.c

```diff
@@ -1952,10 +1952,14 @@ static int simple_varlink_call(const char *option, const char *method) {
                 return log_error_errno(r, "Failed to connect to /run/systemd/journal/io.systemd.journal: %m");
 
         (void) varlink_set_description(link, "journal");
+        (void) varlink_set_relative_timeout(link, 120 * USEC_PER_SEC);
```

Comment:
Now that we know it seems to be working I'm wondering what should happen to the timeout set in the unit file: 0775b9b61120f8be99c and 75f709fbf284041. Before varlink was merged it had been relatively easy to tweak that timeout (in asan_wrapper for example) to adapt the service for almost any environment. I'm not sure I understand how all these timeouts are supposed to interact with each other.

Reply:
By the way, I haven't wrapped my head around varlink yet so it's absolutely possible that I'm missing something :-)